pom.xml

```diff
@@ -300,24 +300,26 @@
         <version.httpunit>1.7.2</version.httpunit>
         <version.io.agroal>1.3</version.io.agroal>
-        <version.io.jaegertracing>0.34.3</version.io.jaegertracing>
+        <version.io.jaegertracing>1.5.0</version.io.jaegertracing>
         <version.io.netty>4.1.58.Final</version.io.netty>
-        <version.io.opentracing>0.31.0</version.io.opentracing>
-        <version.io.opentracing.concurrent>0.2.1</version.io.opentracing.concurrent>
-        <version.io.opentracing.interceptors>0.0.4.1</version.io.opentracing.interceptors>
-        <version.io.opentracing.jaxrs2>0.4.1</version.io.opentracing.jaxrs2>
-        <version.io.opentracing.tracerresolver>0.1.5</version.io.opentracing.tracerresolver>
-        <version.io.opentracing.servlet>0.2.3</version.io.opentracing.servlet>
+        <version.io.opentracing>0.33.0</version.io.opentracing>
+        <version.io.opentracing.concurrent>0.4.0</version.io.opentracing.concurrent>
+        <version.io.opentracing.interceptors>0.1.3</version.io.opentracing.interceptors>
+        <version.io.opentracing.jaxrs2>1.0.0</version.io.opentracing.jaxrs2>
+        <version.io.opentracing.tracerresolver>0.1.8</version.io.opentracing.tracerresolver>
+        <version.io.opentracing.servlet>0.4.1</version.io.opentracing.servlet>
         <version.io.reactivex.rxjava2>2.2.19</version.io.reactivex.rxjava2>
         <version.io.reactivex.rxjava3>3.0.9</version.io.reactivex.rxjava3>
         <version.io.rest-assured>3.0.6</version.io.rest-assured>
-        <version.io.smallrye.smallrye-config>1.6.2</version.io.smallrye.smallrye-config>
-        <version.io.smallrye.smallrye-fault-tolerance>4.3.2</version.io.smallrye.smallrye-fault-tolerance>
-        <version.io.smallrye.smallrye-health>2.2.5</version.io.smallrye.smallrye-health>
-        <version.io.smallrye.smallrye-jwt>2.0.13</version.io.smallrye.smallrye-jwt>
-        <version.io.smallrye.smallrye-metrics>2.4.4</version.io.smallrye.smallrye-metrics>
-        <version.io.smallrye.open-api>2.0.16</version.io.smallrye.open-api>
-        <version.io.smallrye.opentracing>1.3.4</version.io.smallrye.opentracing>
+        <version.io.smallrye.smallrye-common>1.5.0</version.io.smallrye.smallrye-common>
+        <version.io.smallrye.smallrye-config>2.0.2</version.io.smallrye.smallrye-config>
+        <version.io.smallrye.smallrye-fault-tolerance>5.0.0</version.io.smallrye.smallrye-fault-tolerance>
+        <version.io.smallrye.smallrye-health>3.0.0</version.io.smallrye.smallrye-health>
+        <version.io.smallrye.smallrye-jwt>3.0.0</version.io.smallrye.smallrye-jwt>
+        <version.io.smallrye.smallrye-metrics>3.0.1</version.io.smallrye.smallrye-metrics>
+        <version.io.smallrye.open-api>2.1.1</version.io.smallrye.open-api>
+        <version.io.smallrye.opentracing>2.0.0-RC1</version.io.smallrye.opentracing>
+        <version.io.smallrye.smallrye-reactive-converters>2.0.0.RC3</version.io.smallrye.smallrye-reactive-converters>
```

Comment:
@jasondlee / @rhusar this looks to have come in as part of the FP work. Is there a final release?

Reply:
@kabir Final was released 2 weeks ago - opening a PR now.

Reply:
@kabir PR is up https://github.com/wildfly/wildfly/pull/14003 - thanks for noticing this!